scripts/exregional_bias_correction_o3.sh

```diff
@@ -214,39 +205,37 @@ POST_STEP
 
 cp ${DATA}/out/ozone/${yyyy}/*nc ${DATA}/data/bcdata.${yyyymm}/interpolated/ozone/${yyyy}
 
-if [ "${DO_AQM_SAVE_AIRNOW_HIST}" = "TRUE" ]; then
```

Comment:
@JianpingHuang-NOAA, if you remove this if-statement line, this script will work only for you. Please do not remove this if-statement line.